.github/workflows/AirspeedVelocity.yml

```
@@ -0,0 +1,78 @@
+name: Benchmark a Pull Request
+
+on:
+  pull_request_target:
```

```suggestion
  push:
```
I think `pull_request_target` is not triggered for a `push` event, cf. https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target. I think running the bechmarks for every push is reasonable. We do the same for the CI runs. If I understand correctly, the comment from github-actions will be updated automatically after each new push. Let's see if this triggers the Action.